### ISSUE TITLE
Feature request: CAPTCHA solver to renew cookie everyday

### ISSUE THREAD

**Issue by armadill** [maximerenou/php-bing-ai#13]
today when trying make a image it show Generating... Generation failed

**Comment by maximerenou** [comment]
Hello,

Did you send a message to Bing Chat (via bing.com) before using your cookie? I just discovered they added a CAPTCHA, so we must validate our cookie on bing.com before using it out there!

**Comment by armadill** [comment]
yes,, everyday bing make validation. when i use cookie bing send empty respon and have to validate from bing via edge and the cookie works again.. 
and this happens every day, how to pass this validation ?
this applies to chat as well

**Comment by maximerenou** [comment]
Well, that's a bummer!

I suggest periodically launching a (headless?) controlled browser that visits bing.com/chat with the same cookie. It may suffice to keep it valid for automations like this package.

I'm open to PRs if anybody wants to build a bridge with such a thing.

Greetings
